source/includes/fact-wiredtiger-cachesizegb.rst

```diff
@@ -0,0 +1,12 @@
+The default value for this setting is optimized for a single :program:`mongod`
```

Comment:
would change "this setting" to the specific setting (a little redundant, but since it is an include file, it's kind of necessary).

Is it optimized?  That claim is one of those that needs to be verified.  If true, then fine.  If not, update word choice.